payloads/src/payload/payload.rs

```diff
@@ -83,9 +88,36 @@ pub enum stype {
     PAYLOAD_SEGMENT_BSS = 0x42535320,
     PAYLOAD_SEGMENT_PARAMS = 0x50415241,
     PAYLOAD_SEGMENT_ENTRY = 0x454E5452,
+    PAYLOAD_SEGMENT_DTB = 0x44544220,
+    PAYLOAD_SEGMENT_BAD = 0xFFFFFFFF,
+    CBFS_SEGMENT_CODE = 0x45444F43,
+    CBFS_SEGMENT_DATA = 0x41544144,
+    CBFS_SEGMENT_BSS = 0x20535342,
+    CBFS_SEGMENT_PARAMS = 0x41524150,
+    CBFS_SEGMENT_ENTRY = 0x52544E45,
 }
 
+// I give up.
+impl From<u32> for stype {
+    fn from(s: u32) -> Self {
+        match s {
```

Comment:
Maybe do what they suggest here? https://enodev.fr/posts/rusticity-convert-an-integer-to-an-enum.html